locale/id/LC_MESSAGES/django.po

```diff
@@ -24,7 +24,7 @@ msgstr "perempuan"
 
 #: gem/constants.py:6
 msgid "don't want to answer"
```

Comment:
@miltontony maybe a great goal for automation? E.g.: add some functionality to pootle to update from `develop`